installer/Dockerfile

```diff
@@ -87,7 +87,6 @@ RUN sed -i "s/%PHP%/${PHP}/g" /app/services.sh
 WORKDIR /var/www
 
 RUN git clone "${WIKIJUMP_REPO}" "${WIKIJUMP_REPO_DIR}"
-RUN chmod -R 777 "${WIKIJUMP_REPO_DIR}/tmp/smarty_templates_c"
```

Comment:
I'm pretty sure bluesoul said that Wikidot will refuse to run if `tmp/smarty_templates_c` doesn't have permissions of `777`

Reply:
Turns out that is correct; fixed in 34eb5643c11b591c304bd4fe0b28198c26ae988b